scripts/rebuild-testbench.sh

```diff
@@ -5,10 +5,15 @@
 # stop on most errors
 set -e
 
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+SOF_REPO=$(dirname "$SCRIPT_DIR")
+BUILD_TESTBENCH_DIR="$SOF_REPO"/tools/testbench
+
 # Defaults
 BUILD_TYPE=native
 BUILD_DIR_NAME=build_testbench
 BUILD_TARGET=install
+: "${SOF_AFL:=$HOME/sof/work/AFL/afl-gcc}"
```

Comment:
This is strange but it has been always in this script. Please add comment if you know what for it is.

Reply:
This provides a value if none is found in the environment.

The syntax is cryptic but it's standard and documented, it's not even a bashism:
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02